examples/gno.land/r/gnoland/valopers/valopers.gno

```diff
@@ -141,22 +140,27 @@ func isValoper(address std.Address) bool {
 // This function is meant to serve as a helper
 // for generating the govdao proposal
 func GovDAOProposal(address std.Address) {
-	valoper := GetByAddr(address)
-	if !valoper.Active {
-		panic(errValoperInactive)
-	}
+	var (
+		valoper     = GetByAddr(address)
+		votingPower = uint64(1)
+	)
 
 	// Make sure the valoper is the caller
 	if std.GetOrigCaller() != address {
```

Comment:
```suggestion
	if std.GetOrigCaller() != address && !isDaoMember(address) {
```

BTW, I believe we should also allow DAO members to create proposals using this template.
Edit: Changed my mind. Let's try it this way (current implem) and we can expect DAO members to use `maketx run` for exceptions.